### PR TITLE
Feedback data changes

### PR DESCRIPTION
**Pull request by tej99**
Accommodate for changes made in sdx-validate for feedback schema. 

## Checklist
  - [ ] CHANGELOG.md updated? (if required)

### CODE REVIEW

**Comment by tej99** [comment]
[(3) Add validation for feedback survey](https://trello.com/c/sggwJuwT/146-3-add-validation-for-feedback-survey)

File: server.py [hunk 1]
```diff
@@ -208,8 +208,8 @@ def save_response(bound_logger, survey_response):
 def save_feedback_response(bound_logger, survey_feedback_response):
     bound_logger.info("Saving feedback response")
     invalid = survey_feedback_response.get("invalid")
-    survey = survey_feedback_response.get("survey_type")
-    period = survey_feedback_response.get("survey_period")
+    survey = survey_feedback_response.get("associated_survey")
+    period = survey_feedback_response.get("collection").get("period")
```
Comment (JamesGardiner) [inline server.py]: If 'collection' is not a key you will raise an AttributeError. Is this the behaviour you want?

Comment (tej99) [reply server.py]: Oops missed that, thanks

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-### Unreleased
+### 2.1.0 2017-07-25
```
Comment (necrophonic) [inline CHANGELOG.md:1]: Why is the version being updated in a feature/fix branch? The version should be getting updated as part of the prep and tag for release.

Comment (tej99) [reply CHANGELOG.md:1]: For some reason when I do 'git rebase master' the commits that this branch was behind on, compared to master, are shown as part of this pull request.

**Comment by necrophonic** [comment]
About to re-open a fresh pull to sort merge commits

**Pull request closed without merging**